bigtable/quickstart/main.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START bigtable_quickstart]
+import argparse
+
+from google.cloud import bigtable
+
+
+def main(project_id, instance_id, table_id):
+    # Creates a Bigtable client
+    client = bigtable.Client(project=project_id)
+
+    # Connect to an existing instance:my-bigtable-instance
+    instance = client.instance(instance_id)
+
+    # Connect to an existing table:my-table
+    table = instance.table(table_id)
+
+    key = 'r1'
+    row = table.read_row(key.encode('utf-8'))
+
+    column_family_id = 'cf1'
+    column_id = 'c1'.encode('utf-8')
+    value = row.cells[column_family_id][column_id][0].value.decode('utf-8')
+
+    print('Row key: {}\nData: {}'.format(key, value))
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
```

Comment:
In my opinion, the argparse stuff might be a distraction from the key pieces of the quickstart, but we haven't done UX studies on that as far as I know, so I'll allow it. :-)

Reply:
That is a good idea, but how would you suggest pulling in the information? Env variables? Also I wasn't sure if there is an easy way to customize the documentation since so much of it seems to autogenerated and especially using this argument parser as part of that generation.

Reply:
Take a look at https://github.com/GoogleCloudPlatform/golang-samples/blob/master/bigtable/quickstart/main.go
There's a way to do it with region tags so that you can see only the default arguments in the docs, yet offer command line arg support if you were to run it from the command line.

Reply:
Yeah, it's okay as-is. I've seen other quickstarts hard-code a project ID but that comes with its own problems.

Reply:
Ah, yeah that Go example looks like a good compromise.